Backend/TimeTracker.Api/DTOs/DateRangeDTO.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace TimeTracker.Api.DTOs
+{
+    public class DateRangeDTO
+    { 
+        public DateTime startDate { get; set; }
```

Comment:
Make StartDate and EndDate Capital letters

Reply:
Wait ya what am I doing.. been writing too much js all-day